peers/app_request_network.go

```diff
@@ -212,12 +212,12 @@ type ConnectedCanonicalValidators struct {
 	ConnectedWeight       uint64
 	TotalValidatorWeight  uint64
 	ValidatorSet          []*warp.Validator
-	nodeValidatorIndexMap map[ids.NodeID]int
+	NodeValidatorIndexMap map[ids.NodeID]int
```

Comment:
I'm not sure that I love this struct having all of it's fields exported instead of using getters/setters but every other field is already exported so not a blocker for this PR